terra2/assetlist.json

```diff
@@ -366,6 +366,33 @@
         }
       ]
     },
+    {
+      "description": "ClonEliteFund invests in Cosmos ecosystem projects, focusing on scalability, security, and interoperability to maximize returns.",
+      "type_asset": "cw20",
+      "address": "terra164ssz60yvsxey0ku9mtcaegdeyxwzuwwqyrp238nvflwqve0pvxsra7fa2",
+      "denom_units": [
+        {
+          "denom": "cw20:terra164ssz60yvsxey0ku9mtcaegdeyxwzuwwqyrp238nvflwqve0pvxsra7fa2",
+          "exponent": 0
+        },
+        {
+          "denom": "CLON",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra164ssz60yvsxey0ku9mtcaegdeyxwzuwwqyrp238nvflwqve0pvxsra7fa2",
+      "name": "CLON",
+      "display": "CLON",
+      "symbol": "CLON",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/clon1.png"
```

Comment:
Can you please remove the extra padding from the image? The logo will appear tiny with all that padding